WalkingChallenge/LocalizedStrings.swift

```diff
@@ -92,6 +92,17 @@ struct Strings {
     static let joinChallenge: String =
         NSLocalizedString("Join Challenge", comment: "Join Challenge")
   }
+
+  struct Notifications {
+    static let title =
+      NSLocalizedString("Notifications", comment: "Notifications")
+    static let notificationCellTitleLabelPlaceholderTextFormat: String =
+        NSLocalizedString("%@ %@ has joined your team!",
```

Comment:
Do we really want to display this as `<FirstName> <LastName>`?  Do certain regions not have alternative display formats?

Reply:
I'm not sure. Maybe we can leave it until we get the response data?